flow_test.go

```diff
@@ -520,3 +521,44 @@ func BenchmarkFlowUnmarshal(b *testing.B) {
 		_ = f.unmarshal(&iad)
 	}
 }
+
+// TestFlowsMemoryUsage is used to measure memory usage when storing a large
+// number of Flow objects. We don't use a Benchmark for this, because we want to
+// know memory usage *after* garbage collection has run (i.e., we want to
+// exclude allocations for temporary objects when unmarshaling).
+func TestFlowsMemoryUsage(t *testing.T) {
```

Comment:
I don't think this test adds much value, could you remove this commit?

Benchmarks do provide the highest signal to noise ratio when it comes to performance and allocations. Note that a single `runtime.GC()` is not guaranteed to free all unreferenced objects. Objects with finalizers first need to have these run, and are punted to a future cycle. This process is not deterministic, and will depend on the order the `testing` package decides to execute the tests in. Only consider this a rough approximation at best.

Reply:
Yes I don't mind removing this. Benchmarks are not too great at indicating how much memory a long-running program may be consuming.